collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java

```diff
@@ -72,28 +69,17 @@
         return failedResetActions;
     }
 
-    private void resetPreferences(Context context) {
-        boolean clearedDefaultPreferences = PreferenceManager
-                .getDefaultSharedPreferences(context)
-                .edit()
-                .clear()
-                .commit();
-
-        PreferenceManager.setDefaultValues(context, R.xml.preferences, true);
-
-        boolean clearedAdminPreferences = context
-                .getSharedPreferences(AdminPreferencesActivity.ADMIN_PREFERENCES, 0)
-                .edit()
-                .clear()
-                .commit();
+    private void resetPreferences() {
+        GeneralSharedPreferences.getInstance().loadDefaultValues();
+        AdminSharedPreferences.getInstance().loadDefaultValues();
```

Comment:
Would prefer to see these injected into the ResetUtility instance so they can be mocked for testing but you don't need to change it now.